modules/mod_ginger_foundation/lib/css/src/blocks/_list.scss

```diff
@@ -156,23 +156,9 @@
 }
 
 .list__more {
+    @extend %btn--primary;
     display: block;
-    @extend %btn;
     margin: 0 auto;
-    background: $mainThemeColor;
-    color: $white;
-    border-radius: $interfaceRadius;
-
-    &:hover,
-    &:active,
-    &:focus {
-        @extend %btn--primary:hover;
-    }
-
-    + .list__more {
-        margin-top: 20px;
-        background: lighten($mainThemeColor, 20);
-    }
```

Comment:
Note from review with Colin: .list__more should stay. And the border-radius of %btn (18px) is different from $interfaceRadius (4px).